### PR TITLE
AAP-30912 Devtools - Add roles collections chapter

### PR DESCRIPTION
**Pull request by ariordan-redhat**
AAP-30912  Add roles collections chapter to the Devtools guide.
Affects `titles/develop-automation-content`

Build preview here: https://ariordan-redhat.github.io/aap-builds/develop-automation-content-aap-30912-2024-09-27.html

### CODE REVIEW

File: downstream/modules/devtools/proc-devtools-scaffold-roles-collection.adoc [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+the destination directory path is relative to the container, not a path in your local system.
+To discover the current directory name in the container, run the pwd command in a terminal in {VSCode}.
+If the current directory in the container is `workspaces`, enter `workspaces/<current_project>/collections`.
+*** If you are using a locally installed version of Ansible Dev tools,
+enter the full path to the directory, for example `/user/<username>/path/to/<collection_directory>`.
+. Click btn:[Create].
+
+.Verification
+
+After the collection has been created, the following message appears in the *Logs* pane of the *Create Ansible collection* tab.
```
Comment (g-murray) [inline downstream/modules/devtools/proc-devtools-scaffold-roles-collection.adoc:39]: "After you have created the collection"?

File: downstream/modules/devtools/proc-devtools-use-roles-collections-aap.adoc [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+The following procedure describes the workflow to add a collection to an {ExecEnvShort}.
+Refer to
+link:{URLBuilder}/assembly-publishing-exec-env#proc-customize-ee-image[Customizing an existing automation executions environment image]
+in the _{TitleBuilder}_ guide for the commands to execute these steps.
+
+. You can pull an {ExecEnvShort} base image from {HubName},
+or you can add your collection to your own custom {ExecEnvShort}.
+. Add the collections that you want to include in the {ExecEnvShort}.
+. Build the new {ExecEnvShort}.
+. Verify that the collections are installed in the {ExecEnvShort}.
```
Comment (g-murray) [inline downstream/modules/devtools/proc-devtools-use-roles-collections-aap.adoc:18]: "Verify you installed the collections in the {ExecEnvShort}"

**Review by g-murray — APPROVED** [review]
LGTM, just two comments on potential rewording

**Review by annibond — APPROVED** [review]
lgtm!

**Pull request merged**